dvc/logger.py

```diff
@@ -114,11 +114,11 @@ def _walk_exc(self, exc_info):
         tb_list = [tb]
 
         # NOTE: parsing chained exceptions. See dvc/exceptions.py for more info
-        while hasattr(exc, "cause") and exc.cause:
-            exc_list.append(str(exc.cause))
+        while hasattr(exc, "__cause__") and exc.__cause__:
```

Comment:
Should we look into `exc.__context__`? BTW, every exception has both `__cause__` and `__context__`, which might be `None` - no need to use `hasattr()`.